ops/pebble.py

```diff
@@ -352,6 +352,18 @@ class _NotProvidedFlag:
 _not_provided = _NotProvidedFlag()
 
 
+class _Unknown:
```

Comment:
I think we should avoid this and just use `'unknown'` in the enum definition. If Pebble does have "unknown" (`FileType` already has an "unknown" value), nothing bad happens -- it's just that the caller can't distinguish between the two, which doesn't seem like it would be an issue in practice. There's also the warning log for debugging.

Reply:
Note that the warning log won't be emitted if Pebble has "unknown" and "unknown" is in the `enum`. If that's ok, then I'll remove the sentinel class. I hadn't really thought about what I'd do in the case of `FileType.UNKNOWN`, so having it always be the string "unknown" makes that easier.